railties/lib/rails/api/task.rb

```diff
@@ -9,14 +10,21 @@ class Task < RDoc::Task
             README.rdoc
             lib/active_support/**/*.rb
           ),
-          exclude: "lib/active_support/vendor/*"
+          exclude: [
+            'lib/active_support/i18n_railtie.rb',
```

Comment:
Unless I'm missing something, there's nothing for the final user here, this file just eases the set up process.